test/functional/cmdLineTests/criu/src/org/openj9/criu/EnvVarFileTest.java

```diff
@@ -407,13 +407,15 @@ static void envVarFileTest16() {
 		System.out.println("Post-checkpoint");
 	}
 
-	static void testCheckTransitionToDebugInterpreterWithEnvVarFile() {
-		String optionsContents = RESTORE_ENV_VAR + "=-XX:+DebugInterpreter";
+	static void envVarFileTest17() {
+		String optionsContents = RESTORE_ENV_VAR + "=-XX:+DebugInterpreter\n";
```

Comment:
If the newline was needed here, it should have been added to `OptionsFileTest.java` on line 339 as well.

Reply:
Other tests in this file include the new line at the end of this string. That is not the case for `OptionsFileTest.java`.

Reply:
Only some of the uses in this file include the newline.